src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.SnapshotManager.cs

```diff
@@ -115,9 +112,15 @@ internal SnapshotManager ToManagerAndFree()
                     Debug.Assert(_symbolToSlot.Count == _walkerStates.Count);
                     Debug.Assert(_symbolToSlot.Count > 0);
                     _symbolToSlot.Free();
+                    ArrayBuilder<(int position, Snapshot snapshot)> builder = ArrayBuilder<(int, Snapshot)>.GetInstance();
```

Comment:
>GetInstance(); [](start = 108, length = 14)

Can we estimate the size?

Reply:
We can get the exact count, I'll pass it through

---
In reply to: [295557199](https://github.com/dotnet/roslyn/pull/36563#discussion_r295557199) [](ancestors = 295557199)

Reply:
No longer relevant.

---
In reply to: [295557699](https://github.com/dotnet/roslyn/pull/36563#discussion_r295557699) [](ancestors = 295557699,295557199)